cluster/gce/container-linux/configure-helper.sh

```diff
@@ -82,6 +84,22 @@ function get-calico-typha-cpu {
   echo "${typha_cpu}"
 }
 
+# Prepare node labels specifically for master or node.
+function prepare-node-labels {
```

Comment:
move this to config-default.sh and config-test.sh.

Reply:
I copied comments to https://github.com/kubernetes/kubernetes/pull/50705/files#r135171008, let's iterate discussion there.